proto/accesscontrol/genesis.proto

```diff
@@ -0,0 +1,28 @@
+syntax = "proto3";
+package seiprotocol.seichain.accesscontrol;
+
+import "gogoproto/gogo.proto";
+import "accesscontrol/constants.proto";
+
+option go_package = "github.com/sei-protocol/sei-chain/x/accesscontrol/types";
+
+message GenesisState {
+    Params params = 1 [(gogoproto.nullable) = false];
+}
+
+message MessageDependencyMapping {
+    MessageType message_type = 1;
+    Module module_name = 2;
+    repeated string resource_dependencies = 3;
+}
```

Comment:
I was going to use a map of <string, list>  for this but it seems like it's not very well supported with protos

https://stackoverflow.com/questions/38886789/protobuf3-how-to-describe-map-of-repeated-string